src/model.coffee

```diff
@@ -51,6 +56,13 @@ class window.Brainstem.Model extends Backbone.Model
 
 
   #
+  # Init
+
+  constructor: (options = {}) ->
+    super
+    @storageManager = StorageManager.get()
```

Comment:
I would like to avoid any sort of globals outside of specs and the bundled gem output.